e2e/test/scenarios/admin/settings/license-and-billing.cy.spec.js

```diff
@@ -0,0 +1,104 @@
+import { restore, describeEE, setTokenFeatures } from "e2e/support/helpers";
+
+const HOSTING_FEATURE_KEY = "hosting";
+const STORE_MANAGED_FEATURE_KEY = "metabase-store-managed";
+const NO_UPSELL_FEATURE_HEY = "no-upsell";
+
+// mocks data the will be returned by enterprise useLicense hook
+const mockBillingTokenFeatures = features => {
+  return cy.intercept("GET", "/api/premium-features/token/status", {
+    "valid-thru": "2099-12-31T12:00:00",
+    valid: true,
+    trial: false,
+    features,
+    status: "something",
+  });
+};
+
+describe("scenarios > admin > license and billing", () => {
+  beforeEach(() => {
+    restore();
+    cy.signInAsAdmin();
+  });
+
+  describeEE("store info", () => {
+    it("should show the user a link to the store for an unlincensed enterprise instance", () => {
+      cy.visit("/admin/settings/license");
+      cy.findByTestId("license-and-billing-content")
+        .findByText("Go to the Metabase Store")
+        .should("have.prop", "tagName", "A");
+    });
+
+    it("should show the user store info for an self-hosted instance managed by the store", () => {
+      setTokenFeatures("all");
+      mockBillingTokenFeatures([
+        STORE_MANAGED_FEATURE_KEY,
+        NO_UPSELL_FEATURE_HEY,
+      ]);
+
+      // TODO: remove once endpoint returns values as expected
+      cy.intercept("GET", "/api/ee/billing", {
```

Comment:
👍  I'm not sure this is very valuable with the intercept in place, but it should be easy to update once we've got the backend working as expected.

Reply:
fixed!